src/ompl/geometric/planners/rrt/RRTstar.h

```diff
@@ -256,14 +322,26 @@ namespace ompl
             /** \brief Deletes (frees memory) the motion and its children motions. */
             void deleteBranch(Motion *motion);
 
-            /** \brief Computes the Cost To Go heuristically as the cost to come from start to motion plus
-                 the cost to go from motion to goal. If \e shortest is true, the estimated cost to come
-                 start-motion is given. Otherwise, this cost to come is the current motion cost. */
-            base::Cost costToGo(const Motion *motion, const bool shortest = true) const;
+            /** \brief Computes the solution cost heuristically as the cost to come from start to the motion plus
+                 the cost to go from the motion to the goal. If \e estimate is true, a heuristic estimate of the
+                 cost to come is used; otherwise, the current cost to come is used. */
+            base::Cost solutionHeuristic(const Motion *motion, const bool estimate = true) const;
+
+            /** \brief Computes the cost-to-go heuristically for goal states and goal regions using the motionCost given by the optimization objective.*/
+            base::Cost defaultCostToGoHeuristic(const base::State *state, const base::Goal *goal) const;
+
+            /** \brief Count the number of vertices that lay outside the informed subset and store the number in numVerticesWorseThanSoln_.
+            This is a sort of virtual pruning that avoids the problem of removing promising descendents simply
+            because of the quality of their ancestors.
+            \todo Replace with an actually pruning method. */
+            void countNumberOfVerticesWorseThanSoln();
```

Comment:
If I were to rewrite the pruning function as described above, this function could be removed. It's like a sort of *virtual*-prune.